src/pudl/io_managers.py

```diff
@@ -568,12 +568,13 @@ def __init__(self, base_path: UPath, schema: pa.Schema) -> None:
         super().__init__(base_path=base_path)
         self.schema = schema
 
-    def dump_to_path(self, context: OutputContext, obj: pd.DataFrame, path: UPath):
+    def dump_to_path(self, context: OutputContext, obj: dd.DataFrame, path: UPath):
         """Write dataframe to parquet file."""
         raise NotImplementedError("This IO Manager doesn't support writing data.")
 
-    def load_from_path(self, context: InputContext, path: UPath) -> pd.DataFrame:
+    def load_from_path(self, context: InputContext, path: UPath) -> dd.DataFrame:
         """Load a directory of parquet files to a dask dataframe."""
+        logger.info(f"Reading parquet file from {path}")
         return dd.read_parquet(
             path.parent,
```

Comment:
I think the IO Manager should read the monolithic file now that the partitioned version is optional.

Reply:
Good call, fixed